fw/http.c

```diff
@@ -7445,14 +7491,14 @@ tfw_cfgop_cleanup_brange_##name(TfwCfgSpec *cs)				\
 	tfw_init_custom_##name(NULL);					\
 }
 
-TFW_HTTP_CFG_CUSTOM_BRANGE(uri);
-TFW_HTTP_CFG_CUSTOM_BRANGE(token);
-TFW_HTTP_CFG_CUSTOM_BRANGE(qetoken);
-TFW_HTTP_CFG_CUSTOM_BRANGE(nctl);
-TFW_HTTP_CFG_CUSTOM_BRANGE(ctext_vchar);
-TFW_HTTP_CFG_CUSTOM_BRANGE(xff);
-TFW_HTTP_CFG_CUSTOM_BRANGE(cookie);
-TFW_HTTP_CFG_CUSTOM_BRANGE(etag);
+TFW_HTTP_CFG_CUSTOM_BRANGE(uri, "0x00-0x20");
+TFW_HTTP_CFG_CUSTOM_BRANGE(token, "0x00-0x20 0x2c 0x3b");
+TFW_HTTP_CFG_CUSTOM_BRANGE(qetoken, "0x00-0x20 0x2c");
```

Comment:
@RomanBelozerov Why do we disallow `0x2c` legit to qetoken? However for token we disallow `,;` but allow potentially "malicious" characters `"\:/`?

RFC qetoken:
>   quoted-string  = DQUOTE *( qdtext / quoted-pair ) DQUOTE
>   qdtext         = HTAB / SP / %x21 / %x23-5B / %x5D-7E / obs-text

Reply:
after call discussion we decided not to restrict the list of characters . I added a list of characters that break the parser

Reply:
@RomanBelozerov So this issue is canceled?

Reply:
@kingluo Please, change `qetoken` disallowed characters accordingly to RFC. `HTAB / SP / %x21 / %x23-5B / %x5D-7E/ obs-text` must not be disallowed.